fighthealthinsurance/settings.py

```diff
@@ -356,8 +374,30 @@ class TestSync(Dev):
 
 
 class Prod(Base):
+    sentry_endpoint = os.getenv("SENTRY_ENDPOINT")
+
+    if sentry_endpoint:
+
+        import sentry_sdk
+
+        sentry_sdk.init(
+            dsn=sentry_endpoint,
+            # Set traces_sample_rate to 1.0 to capture 100%
+            # of transactions for tracing.
+            traces_sample_rate=1.0,
+            _experiments={
+                # Set continuous_profiling_auto_start to True
+                # to automatically start the profiler on when
+                # possible.
+                "continuous_profiling_auto_start": True,
+            },
+        )
+
     DEBUG = False
 
+    # Different fido server for production
+    FIDO_SERVER_ID = "fighthealthinsurance.com"  # Server rp id for FIDO2, it is the full domain of your project
```

Comment:
Maybe this should be grabbed from an environment variable so that we can use different domains in prod/dev more easily?

Reply:
oooh interesting